### DISCUSSION TITLE
Skywalking's feature 'ignore path' can only ignore the request URL?

### DISCUSSION THREAD

**Discussion by xunyao4dev** [apache/skywalking#12501] [Q&A] [answered]
It seems that this feature can only ignore request URL, eg. 'GET:/orders/**', but I want to ignore the some parts of the trace tree, for example bellow：
<img width="375" alt="94eafb574b0809048810e0807624fbb6" src="https://github.com/user-attachments/assets/0f0ae021-b393-4ee1-8516-e4629b2f819a">
 I wonder how to config this? or I need develop a custom plugin?

**Comment by wu-sheng** [comment, marked as answer]
https://github.com/apache/skywalking/discussions/8767#discussioncomment-10220120

Duplicated.
